vico/views/src/main/java/com/patrykandpatrick/vico/views/common/ChartView.kt

```diff
@@ -199,6 +210,21 @@ constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
     @Suppress("UNNECESSARY_SAFE_CALL")
     measuringContext?.isLtr = layoutDirection == LAYOUT_DIRECTION_LTR
   }
+
+  protected fun MotionEvent.translateOrReject(): Boolean {
+    val action = action and MotionEvent.ACTION_MASK
+    if (action == MotionEvent.ACTION_DOWN) {
+      acceptMotionEvents =
+        (0..pointerCount - 1).all { index ->
```

Comment:
What about using `until` instead here?
```suggestion
        (0 until pointerCount).all { index ->
```

Reply:
Better yet, we can use `..<`. I’ve switched to it in [`f6ab1c3`](https://github.com/patrykandpatrick/vico/pull/1223/commits/f6ab1c329b5ef2f9a17525a44d2329f4404a2997).